### PR TITLE
Add checklist for before submitting an issue on GitHub

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -94,7 +94,13 @@ If you're not familiar with the terminal and prefer a graphical user interface,
 
 ## Questions and support
 
-All fastlane related questions, problems and feature ideas should be posted as [GitHub issue](https://github.com/fastlane/fastlane/issues/new). This way, other users can search existing discussions, and everybody can jump in.
+Before submitting a new GitHub issue, please make sure to
+
+- Check out [docs.fastlane.tools](https://docs.fastlane.tools)
+- Check out the README pages on [this repo](https://github.com/fastlane/fastlane)
```
Comment (ohayon) [inline docs/index.md]: should this point to fastlane/docs?

File: docs/index.md [hunk 2]
```diff
@@ -94,7 +94,13 @@ If you're not familiar with the terminal and prefer a graphical user interface,
...
+- Search for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)
```
Comment (ohayon) [inline docs/index.md]: same as above?

File: docs/index.md (hunk 1 above)
Comment (KrauseFx) [inline docs/index.md]: The docs is the first item, this points to the GitHub repo, as we still have tons of docs on there :)

Comment (ohayon) [inline docs/index.md]: 👍

**Pull request merged**